apis/python/src/tiledbsoma/options/_soma_tiledb_context.py

```diff
@@ -17,11 +17,21 @@
 import tiledb
 
 from .. import pytiledbsoma as clib
-from .._general_utilities import get_implementation_version
+from .._general_utilities import get_release_version
 from .._types import OpenTimestamp
 from .._util import ms_to_datetime, to_timestamp_ms
 
 
+def _tiledb_ctx_deprecation() -> None:
+    assert get_release_version() < (1, 14)
+    warnings.warn(
+        "tiledb_ctx is now deprecated for removal in 1.14. "
+        "Use tiledb_config instead by passing "
+        "SOMATileDBContext(tiledb_config=ctx.config()).",
```

Comment:
Won't this still require `tiledb` (i.e. tiledb-py)?

I believe we'll need them to use `ctx.config().dict()`, IIUC.